src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment: { name, email, body } }) => (
```

Comment:
It's a good practice to destructure your props inside the component body, not in the argument. This makes it easier to add type annotations or default values later.